web/packages/teleterm/src/ui/AppInitializer.tsx

```diff
@@ -1,20 +1,23 @@
-import React, { FC, useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import styled from 'styled-components';
+import { Indicator } from 'design';
 
 import { useAppContext } from './appContextProvider';
 import { initUi } from './initUi';
+import ModalsHost from './ModalsHost';
+import { LayoutManager } from './LayoutManager';
 
-export const AppInitializer: FC = props => {
+export const AppInitializer = () => {
   const ctx = useAppContext();
-  const [isUiVisible, setIsUiVisible] = useState(false);
+  const [isUiReady, setIsUiReady] = useState(false);
 
   const initializeApp = useCallback(async () => {
     try {
       await ctx.init();
-      setIsUiVisible(true);
       await initUi(ctx);
+      setIsUiReady(true);
     } catch (error) {
-      setIsUiVisible(true);
+      setIsUiReady(true);
```

Comment:
I considered doing this in `finally` block once but I guess the order of execution between this an `notifyError` would have been different and I didn't want to mess with that.